src/balatro_gym/cards/interfaces.py

```diff
@@ -262,6 +274,16 @@ def get_multiplication(self) -> int:
             return int(self.enhancement.get_multiplication())
         return 1
 
+    def get_scored_money(self) -> int:
+        if isinstance(self.enhancement, HasMoney):
```

Comment:
Note: initially I wanted to have the modifiers etc non-optional. That is, by default a `PlayingCard` would be instantiated with a `BaseSeal`, `BaseEnhancement`, and `BaseEdition` that would be 'pass through'. This would make it unnecessary to check if an enhancement exists, or if its of a given type, and could just call `self._enhancement.get_scored_money()` directly.

I'm unsure if this will make it more complicated for `Joker`s to reason about though, for the jokers that care about hand state.

Reply:
I'm good with either version!